src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/RSBridgePeripheral.java

```diff
@@ -0,0 +1,893 @@
+        if (!isAvailable())
+            return notConnected();
+
+        IFluidHandler handler = FluidUtil.getHandlerFromDirection(arguments.getString(1), owner);
+        if (handler == null)
+            handler = FluidUtil.getHandlerFromName(computer, arguments.getString(1));
+
+        if (handler == null)
+            return MethodResult.of(0, "The target tank does not exist. Make sure the bridge is exposed in the computer network. Reach out to our discord or our documentation for help.");
+
+        return exportToTank(arguments, handler);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult importChemical(IComputerAccess computer, IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected();
+
+        IChemicalHandler handler = ChemicalUtil.getHandlerFromDirection(arguments.getString(1), owner);
+        if (handler == null)
+            handler = ChemicalUtil.getHandlerFromName(computer, arguments.getString(1));
+
+        if (handler == null)
+            return MethodResult.of(0, "The target tank does not exist. Make sure the bridge is exposed in the computer network. Reach out to our discord or our documentation for help.");
+
+        return importToRS(arguments, handler);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult exportChemical(IComputerAccess computer, IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected();
+
+        IChemicalHandler handler = ChemicalUtil.getHandlerFromDirection(arguments.getString(1), owner);
+        if (handler == null)
+            handler = ChemicalUtil.getHandlerFromName(computer, arguments.getString(1));
+
+        if (handler == null)
+            return MethodResult.of(0, "The target tank does not exist. Make sure the bridge is exposed in the computer network. Reach out to our discord or our documentation for help.");
+
+        return exportToTank(arguments, handler);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult getStoredEnergy() {
+        if (!isAvailable())
+            return notConnected();
+
+        EnergyNetworkComponent energyComponent = getNetwork().getComponent(EnergyNetworkComponent.class);
+
+        return MethodResult.of(energyComponent.getStored());
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult getEnergyCapacity() {
+        if (!isAvailable())
+            return notConnected();
+
+        EnergyNetworkComponent energyComponent = getNetwork().getComponent(EnergyNetworkComponent.class);
+
+        return MethodResult.of(energyComponent.getCapacity());
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult getEnergyUsage() {
+        if (!isAvailable())
+            return notConnected();
+
+        return MethodResult.of(0);
```

Comment:
is this not available in RS api or is this a TODO thing

Reply:
There is a way in RS but it was not as straightforward so I skipped it. Gonna implement it later.
We need to loop through all nodes, and add the energy usage together